tests/client/test_orion_client.py

```diff
@@ -822,6 +822,11 @@ async def test_create_flow_run_from_deployment_idempotency(orion_client, deploym
 
     assert flow_run_2.id == flow_run_1.id
 
+    flow_run_3 = await orion_client.create_flow_run_from_deployment(
+        deployment.id, idempotency_key="bar"
+    )
+    assert flow_run_3.id != flow_run_1.id
```

Comment:
This is probably redundant if tested below in `models/test_flow_runs.py`

Reply:
Seems worth including to me! Thanks!